src/components/common/layout/header/components/mobile-header/MobileHeader.tsx

```diff
@@ -0,0 +1,185 @@
+                text="Загальне"
+                position={TabItemContentPosition.LEFT}
+                icon={<AcademicCapIcon />}
+                size={TabItemContentSize.SMAll}
+              />
+            </Link>
+            <Link href={'/account?tab=security'} onClick={handleClick}>
+              <TabItem
+                text="Безпека"
+                position={TabItemContentPosition.LEFT}
+                icon={<LockClosedIcon />}
+                size={TabItemContentSize.SMAll}
+              />
+            </Link>
+            <Link href={'/account?tab=group'} onClick={handleClick}>
+              <TabItem
+                text="Група"
+                position={TabItemContentPosition.LEFT}
+                icon={<UsersIcon />}
+                size={TabItemContentSize.SMAll}
+              />
+            </Link>
+          </Box>
+          <HeaderDivider />
+          <HeaderMobileMenu setIsOpened={handleClick} />
+        </Box>
+      </Box>
+    ) : (
+      <Box sx={styles.headerContainer}>
+        <Link href={'/'}>
+          <Box sx={styles.headerLogo}>
+            <img src={`/assets/logo.png`} alt="logo" />
+          </Box>
+        </Link>
+        <Box sx={styles.mobileButton}>
+          <IconButton
+            onClick={handleClick}
+            size={IconButtonSize.MEDIUM}
+            color={IconButtonColor.TRANSPARENT}
+            icon={<BurgerMenu />}
+          />
+        </Box>
+      </Box>
+    );
+  }
+
+  if (!isLoggedIn) {
+    return isOpened ? (
+      <Box>
+        <Box sx={styles.shadow} onClick={handleClick}></Box>
+        <Box sx={styles.headerContainer} style={{ backgroundColor: '#1e1e1e' }}>
+          <Link href={'/'}>
+            <Box sx={styles.headerLogo}>
+              <img src={`/assets/logo.png`} alt="logo" />
+            </Box>
+          </Link>
+          <Box sx={styles.mobileButton}>
+            <CloseButton
+              onClick={handleClick}
+              size={IconButtonSize.MEDIUM}
+              color={IconButtonColor.TRANSPARENT}
+            />
+          </Box>
+        </Box>
+        <Box sx={styles.drop}>
+          <Box sx={styles.loginButtons}>
+            <Box style={{ width: '192px' }}>
+              <Link href={'/register'}>
+                <Button
+                  text="Зареєструватись"
+                  size={ButtonSize.SMALL}
+                  variant={ButtonVariant.OUTLINE}
+                />
+              </Link>
+            </Box>
+            <Box style={{ width: '120px' }}>
+              <Link href={'/login'}>
+                <Button
+                  text="Увійти"
+                  size={ButtonSize.SMALL}
+                  variant={ButtonVariant.FILLED}
+                />
+              </Link>
+            </Box>
+          </Box>
+          <HeaderDivider />
+          <HeaderMobileMenu setIsOpened={handleClick} />
+        </Box>
+      </Box>
+    ) : (
+      <Box sx={styles.headerContainer}>
+        <Link href={'/'}>
```

Comment:
Too much nesting. At least use Box to display image, and if possible put href on this box too